# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../coreutils/disable_man.patch.cross
# Copyright (C) 2017 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./man/local.mk.orig        2016-11-22 21:04:32.000000000 +0100
+++ ./man/local.mk     2017-03-02 08:03:54.122400133 +0100
@@ -19,11 +19,11 @@
 EXTRA_DIST += man/help2man man/dummy-man
 
 ## Graceful degradation for systems lacking perl.
-if HAVE_PERL
-run_help2man = $(PERL) -- $(srcdir)/man/help2man
-else
+#if HAVE_PERL
+#run_help2man = $(PERL) -- $(srcdir)/man/help2man
+#else
 run_help2man = $(SHELL) $(srcdir)/man/dummy-man
-endif
+#endif
 
 man1_MANS = @man1_MANS@
 EXTRA_DIST += $(man1_MANS:.1=.x)
@@ -189,8 +189,8 @@
       && export SOURCE_DATE_EPOCH && $(run_help2man)			\
 		     --source='$(PACKAGE_STRING)'			\
 		     --include=$(srcdir)/man/$$name.x			\
-		     --output=$$t/$$name.1				\
 		     --info-page='\(aq(coreutils) '$$name' invocation\(aq' \
+		     --output=$$t/$$name.1				\
 		     $$t/$$argv$(EXEEXT)				\
 	  && sed \
 	       -e 's|$*\.td/||g' \
